modules/project-structure/01-project-structure.qmd

```diff
@@ -0,0 +1,180 @@
+---
+title: Project structure
+subtitle: 'In this topic, you will learn about the project structure and how to manage your project in R properly.'
+title-block-banner: yes
+engine: knitr
+filters:
+  - webr-teachr
+execute: 
+  echo: false
+---
+
+```{r}
+knitr::opts_chunk$set(
+    fig.width = 8,
+    fig.height = 8,
+    fig.align = "center")
+```
+
+# Paths {#sec-path}
+
+The **paths** are used to identify the location of a file, computer, device, or web page. You could think of it as a direction pointing to the destination.
+
+## What does it look like?
+
+### File path
+
+The path may look different depending on the system you are using. If we want to locate a folder called **learnR**, this is what it may look like.
+
+**Windows:**
+
+`C:\Desktop\learnR\data\penguins.csv`
+
+**MacOS & Linux:**
+
+`/Users/Desktop/learnR/data/penguins.csv`
+
+### Webpage
+
+To locate the webpage, we use the URL (Uniform Resource Locator) to identify any resource on the internet.
+
+`https://learnr.numbat.space/data/penguins.csv`
+
+
+## Abolute Paths {#sec-abs-path}
+
+:::{.callout-note}
+# Absolute path
+
+It is a full location of the file or directory specified, starting from the lowest level.
+:::
+
+:::{.callout-important}
+# Use of absolute path
+
+Absolute paths should never be used in projects, as others will not be able to run the code you provide. This is because absolute paths to a project vary with operating systems and where the project is stored.
+
+In @sec-path, we can see that MacOS and Windows have completely different path structures.
+:::
+
+## Relative Paths {#sec-rela-path}
+
+:::{.callout-note}
+# Relative path
+
+It is the location of a file or directory specified relative to your current working directory (the folder you are currently working on).
+:::
+
+To understand a bit more about the current working directory, let me introduce the `here` package. It allows you to find your base folder or root directory for you to start on. The `here()` function dynamically identifies the root directory based on the user systems. It allows using a relative path where we only need to specify where in your project folder you are trying to point to.
+
+```{r}
+#| echo: true
```

Comment:
I think this example is too meta about writing the article, better to focus on data analysis and demonstrate finding data.